lang/cs/Org.Apache.REEF.Tests/Functional/ReefFunctionalTest.cs

```diff
@@ -173,9 +173,30 @@ protected void ValidateSuccessForLocalRuntime(int numberOfEvaluatorsToClose, int
 
         protected void ValidateMessageSuccessfullyLogged(string message, string testFolder)
         {
-            string[] lines = File.ReadAllLines(GetLogFile(_stdout, testFolder));
-            string[] successIndicators = lines.Where(s => s.Contains(message)).ToArray();
-            Assert.IsTrue(successIndicators.Any());
+            string[] lines = null;
```

Comment:
There is a race condition between file creation and reading. Changed it to be similar to the ValidateSuccessForLocalRuntime